### PR TITLE
♏ Add email scope for Google OAuth

### PR DESCRIPTION
**Pull request by WRadoslaw**
#3566

### CODE REVIEW

File: packages/atlas/src/views/global/YppLandingView/YppAuthorizationModal/YppAuthorizationModal.hooks.tsx [hunk 1]
```diff
@@ -32,7 +32,8 @@ const GOOGLE_AUTH_PARAMS = {
   client_id: GOOGLE_CONSOLE_CLIENT_ID || '',
   response_type: 'code',
   access_type: 'offline',
-  scope: 'https://www.googleapis.com/auth/youtube.readonly  https://www.googleapis.com/auth/userinfo.profile',
+  scope:
+    'https://www.googleapis.com/auth/youtube.readonly  https://www.googleapis.com/auth/userinfo.profile https://www.googleapis.com/auth/userinfo.email',
```
Comment (drillprop) [inline packages/atlas/src/views/global/YppLandingView/YppAuthorizationModal/YppAuthorizationModal.hooks.tsx:36]: make sure we have the email in details form step

Comment (WRadoslaw) [reply packages/atlas/src/views/global/YppLandingView/YppAuthorizationModal/YppAuthorizationModal.hooks.tsx:36]: Tested - works

**Review by drillprop — COMMENTED** [review]
I got a very strange email when I was trying to sign up with one of my youtube channel. `test-channel-5-4553@pages.plusgoogle.com`. This is because I'm using a youtube channel(not a google account) to sign up. It looks like the youtube channel has a different email address.
Is it possible to get an email to the owner of a channel? Could you investigate this?

**Comment by WRadoslaw** [comment]
Gotcha

**Review by drillprop — APPROVED** [review]

**Pull request merged**